tips/api/compound_rules.json

```diff
@@ -9,11 +9,56 @@
     ]
   },
   "2": {
-    "name": "Is 18",
+    "name": "Is 18 of ouder",
     "rules": [
       {
         "type": "rule",
-        "rule": "dateTime($.brp.persoon.geboortedatum) - timeDelta(18, 0, 0, 0, 0, 0) <= now()"
+        "rule": "dateTime($.brp.persoon.geboortedatum) + timeDelta(18, 0, 0, 0, 0, 0) <= now()"
+      }
+    ]
+  },
+  "3": {
+    "name": "Woont in gemeente Amsterdam",
+    "rules": [
+      {
+        "type": "rule",
+        "rule": "$.brp.persoon.mokum is true"
+      }
+    ]
+  },
+  "4": {
+    "name": "Heeft kinderen",
+    "rules": [
+      {
+        "type": "rule",
+        "rule": "len($.brp.kinderen) >= 1"
+      }
+    ]
+  },
+  "5": {
+    "name": "Is ingeschreven in Amsterdam",
+    "rules": [
+      {
+        "type": "rule",
+        "rule" : "$.brp.persoon.gemeentenaamInschrijving is Amsterdam"
```

Comment:
deze regel is effectief hetzelfde als nummer 3
https://github.com/Amsterdam/mijn-mks-api/blob/master/mks/model/stuf_02_04.py#L84

Reply:
correct.